docs/source/implementations/sb_bagging.rst

```diff
@@ -0,0 +1,151 @@
+.. _implementations-sb_bagging:
```

Comment:
The documentation generator gives one warning. Also, and more importantly, your documentation is not linked in the main index files. Please see snippet below.

```
mlfinlab/docs/source/implementations/sb_bagging.rst:135: WARNING: Unexpected indentation.
mlfinlab/docs/source/index.rst:63: WARNING: Duplicate substitution definition name: "vspace".
looking for now-outdated files... none found
pickling environment... done
checking consistency... mlfinlab/docs/source/implementations/sb_bagging.rst: WARNING: document isn't included in any toctree
```